cmd/podman/manifest/create.go

```diff
@@ -20,8 +20,9 @@ var (
 		ValidArgsFunction: common.AutocompleteImages,
 		Example: `podman manifest create mylist:v1.11
   podman manifest create mylist:v1.11 arch-specific-image-to-add
+  podman manifest create mylist:v1.11 arch-specific-image-to-add another-arch-specific-image-to-add
   podman manifest create --all mylist:v1.11 transport:tagged-image-to-add`,
-		Args: cobra.RangeArgs(1, 2),
+		Args: cobra.MinimumNArgs(1),
```

Comment:
I think this should be minimalargs(2)

Reply:
Hey @rhatdan, We currently support creating "empty" manifests, so I think minimalargs(1) is correct

Reply:
I concur with @rsevilla87 :+1: